src/test/scala/cognite/spark/v1/SparkTest.scala

```diff
@@ -58,7 +65,8 @@ trait SparkTest {
   val spark: SparkSession = SparkSession
     .builder()
     .master("local[*]")
-    .config("spark.ui.enabled", "false")
+    .config("spark.ui.enabled", "false") //comment this and use the line below if you need to use Spark UI
+    //.config("spark.ui.port", "4041")
```

Comment:
So that we can access to SparkUI when running test in local

Reply:
4040 is the default, this line isn't needed.